### ISSUE TITLE
Stereo sound

### ISSUE THREAD

**Issue by Lislegaard** [eerimoq/moblin#61]
Would it be possible to support stereo sound?

I have an iPhone 13 mini and when I film with the camera app it is stereo, I would love if this was working in moblin as well for streaming music performances.

**Comment by eerimoq** [comment]
Yeah, but just because not enough users have confirmed that it works. Could move the toggle to Settings -> Audio I guess. See what happens =)

**Comment by Lislegaard** [comment]
Allright cool. I'll chime in how it works after some streams 😊 First little test today was positive. 

Is there also some way to adjust the gain or is this done automatically by the Os?

**Comment by eerimoq** [comment]
Haven't looked into it. Can be implemented in the app. Maybe in OS as well, not sure.
